### PR TITLE
[minor](stats) rename stats related session variable name

### PR DESCRIPTION
**Pull request by Kikyou1997**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Kikyou1997** [comment]
run buildall

**Review by wm1581066 — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 1]
```diff
@@ -1286,10 +1281,16 @@ public void setMaxJoinNumberOfReorder(int maxJoinNumberOfReorder) {
             options = {"none", "sql_cache", "partition_cache"})
     public String testQueryCacheHit = "none";
 
-    @VariableMgr.VarAttr(name = ENABLE_FULL_AUTO_ANALYZE,
+    @VariableMgr.VarAttr(name = ENABLE_AUTO_ANALYZE,
             description = {"该参数控制是否开启自动收集", "Set false to disable auto analyze"},
             flag = VariableMgr.GLOBAL)
-    public boolean enableFullAutoAnalyze = true;
+    public boolean enableAutoAnalyze = false;
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:1287]:
```suggestion
    public boolean enableAutoAnalyze = true;
```

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 2]
```diff
@@ -1260,17 +1260,12 @@ public void setMaxJoinNumberOfReorder(int maxJoinNumberOfReorder) {
...
     public int invertedIndexConjunctionOptThreshold = 1000;
 
-    @VariableMgr.VarAttr(name = FULL_AUTO_ANALYZE_START_TIME, needForward = true, checker = "checkAnalyzeTimeFormat",
-            description = {"该参数定义自动ANALYZE例程的开始时间",
-                    "This parameter defines the start time for the automatic ANALYZE routine."},
-            flag = VariableMgr.GLOBAL)
-    public String fullAutoAnalyzeStartTime = "00:00:00";
 
-    @VariableMgr.VarAttr(name = FULL_AUTO_ANALYZE_END_TIME, needForward = true, checker = "checkAnalyzeTimeFormat",
+    @VariableMgr.VarAttr(name = AUTO_ANALYZE_END_TIME, needForward = true, checker = "checkAnalyzeTimeFormat",
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:1264]: put all auto analyze variable togather

**Comment by Kikyou1997** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Pull request merged**